proto/proto/block_producer.proto

```diff
@@ -52,6 +64,20 @@ message SubmitProvenTransactionResponse {
     fixed32 block_height = 1;
 }
 
+// SUBMIT PROVEN TRANSACTION
+// ================================================================================================
+
+message SubmitProvenBatchResponse {
+    // The node's current block height.
+    fixed32 block_height = 1;
+}
+
+message ProvenBatch {
+    // Encoded using [winter_utils::Serializable] implementation for
+    // [miden_objects::transaction::proven_tx::ProvenTransaction].
+    bytes encoded = 1;
+}
```

Comment:
nit: I would maybe name this `ProvenTransactionBatch` and move it into /types/transaction.proto` file.